src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,169 @@
+package duke;
+
+import exceptions.DukeException;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static Task[] taskList = new Task [100];
+    static int taskIndex=0;
+
+    public static void main(String[] args) {
+
+        greetUser("hi");
+        startChatbot();
+        greetUser("bye");
+    }
+
+    private static void startChatbot() {
+        Scanner input = new Scanner(System.in);
+        String task = "";
+        while(true) {
+            try {
+                task = input.nextLine();
+                System.out.println("    ____________________________________");
+                if (task.equals("bye")) {
+                    break;
+                } else if (task.equals("list")) {
+                    listTasks();
+                } else if (task.startsWith("mark ")) {
+                    markTask(task);
+                } else if (task.startsWith("unmark ")) {
+                    unmarkTask(task);
+                } else if (task.startsWith("todo") || task.startsWith("deadline") || task.startsWith("event")) {
```

Comment:
Avoid complicated expressions. Consider breaking up the steps into different statements.